pkg/config/config.go

```diff
@@ -39,9 +39,10 @@ type Metric struct {
 
 // Strategy is the steps and configuration for rollout.
 type Strategy struct {
-	Steps              []int64
-	HealthCriteria     []Metric
-	HealthOffsetMinute int
+	Steps                 []int64
+	HealthCriteria        []Metric
+	HealthOffsetMinute    int
+	TimeRollForwardMinute int
```

Comment:
var name isn't very descriptive.

TimeBetweenRolloutsMins is better?

Reply:
I renamed it to `TimeBetweenRollForwardsMins`

Reply:
Does this not affect rollback timing?

Reply:
It won't affect rollouts. First failed health check causes rollback (even if min request count criteria is not met).